src/ansi-c/expr2c.cpp

```diff
@@ -1822,8 +1835,24 @@ std::string expr2ct::convert_constant(
     return convert_norep(src, precedence);
   else if(type.id()==ID_bv)
   {
-    // not C
-    dest=id2string(value);
+    // used for _BitInt
+    irep_idt c_type = src.get(ID_C_c_type);
+    if(c_type == ID_c_signed_bitint)
+    {
+      auto as_int = bvrep2integer(value, to_bv_type(type).width(), false);
+      auto width = src.get_int(ID_C_c_bitint_width);
+      auto binary = integer2binary(as_int, width); // drops padding
+      return integer2string(binary2integer(binary, true));
+    }
+    else if(c_type == ID_c_unsigned_bitint)
+    {
+      auto as_int = bvrep2integer(value, to_bv_type(type).width(), false);
+      auto width = src.get_int(ID_C_c_bitint_width);
+      auto binary = integer2binary(as_int, width); // drops padding
+      return integer2string(binary2integer(binary, false));
+    }
+    else
+      return convert_norep(src, precedence);
```

Comment:
I think previously this would handle `__CPROVER_bitvector` in some way?

Reply:
Those map to `signedbv`/`unsignedv`.